libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -155,6 +156,12 @@ static RCInput_RPI rcinDriver;
 static RCInput_Raspilot rcinDriver;
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ZYNQ
 static RCInput_ZYNQ rcinDriver;
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_OCPOC_ZYNQ
+#if OCPOC_DEFAULT_RCIN_SBUS
+static RCInput_SBUS rcinDriver;
+#else
+static RCInput_ZYNQ rcinDriver;
```

Comment:
this is never compiled since OCPOC_DEFAULT_RCIN_SBUS is always 1... maybe remove?

Reply:
See my comment regarding the OCPOC_DEFAULT_RCIN_SBUS define.